### PR TITLE
Update DefaultLayout.tsx

### PR DESCRIPTION
**Pull request by gurjmatharu**
- added margin:auto to main

### CODE REVIEW

File: app/components/Layout/DefaultLayout.tsx [hunk 1]
```diff
@@ -51,11 +51,11 @@ const DefaultLayout: React.FC<Props> = ({
             display: flex;
             flex-direction: column;
           }
-
           main {
             padding: 30px 40px;
             flex-grow: 1;
             background-color: #fafafc;
+            margin: auto;
```
Comment (matthieu-foucault) [inline app/components/Layout/DefaultLayout.tsx:58]: - The `background-color` rule may need to be moved to the `#page-wrap` rule set, as the side margins are white
- The "Contact Form" page ends up being too narrow, see screenshot below. Maybe a margin to the right of the "New Contact"/"Edit Contact" heading will fix it? 
image

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]
Not sure why your e2e test failed, but your branch does need to be updated.
We prefer `git rebase` vs merging the base branch into your branch, to keep the git history readable.

File: app/pages/cif/contact/form/[contactForm].tsx [hunk 2]
```diff
@@ -120,6 +120,9 @@ function ContactFormPage({ preloadedQuery }: RelayProps<{}, ContactFormQuery>) {
           justify-content: space-between;
           align-items: start;
         }
+        header h2 {
+          padding-right: 10px;
+        }
```
Comment (matthieu-foucault) [inline app/pages/cif/contact/form/[contactForm].tsx:125]: Let's add the same thing to the contact view page (`app/pages/cif/contact/[contact].tsx`)

Comment (gurjmatharu) [reply app/pages/cif/contact/form/[contactForm].tsx:125]: Added - see update.

**Review by matthieu-foucault — DISMISSED** [review]

**Review by matthieu-foucault — APPROVED** [review]

**Pull request merged**